test/Microsoft.SqlTools.ServiceLayer.Test.Common/Consts.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace Microsoft.SqlTools.ServiceLayer.Test.Commons
```

Comment:
typo: you've put `Commons` instead of `Common` which causes unnecessary using statements being added. Also - why isn't this just merged with the Constants file?